assets/scss/header.scss

```diff
@@ -4,18 +4,162 @@
     max-width: 100%;
 
     li {
-      @extend .govuk-header__navigation-item;
 
-      &.current-menu-item {
-        @extend .govuk-header__navigation-item--active
-      }
+      @extend .govuk-header__navigation-item;
 
       a {
         @extend .govuk-header__link;
         height: 100%;
         padding: 8px 0;
+
+        @media (min-width: 48.0625em) {
+          height: 40px;
+        }
+      }
+
+      &.menu-item-has-children {
+
+        > a {
+          &:before {
+            content: "";
+            border: solid govuk-colour("white");
+            border-width: 0 3px 3px 0;
+            display: block;
+            padding: 3px;
+            transform: rotate(45deg);
+            -webkit-transform: rotate(45deg);
+            float: left;
+            margin: 3px 10px 0 5px;
+
+            @media (min-width: 48.0625em) {
+              content: none;
+            }
+          }
+
+          &:focus {
+            &:before {
+              border: solid govuk-colour("black");
+              border-width: 0 3px 3px 0;
+              padding: 3px;
```

Comment:
Are these needed?  They are the same as lines 26 and 28.

Reply:
Go suggestion. i think i change line 41 to border-color then i wont need the other lines.